README.md

```diff
@@ -130,32 +138,38 @@ The following snippet demonstrates how to create a small data curation pipeline
 dataset = download_common_crawl("/datasets/common_crawl/", "2021-04", "2021-10", url_limit=10)
 # Build your pipeline
 curation_pipeline = Sequential([
+  # Fix unicode
   Modify(UnicodeReformatter()),
+  # Discard short records
   ScoreFilter(WordCountFilter(min_words=80)),
+  # Discard low-quality records
   ScoreFilter(FastTextQualityFilter(model_path="model.bin")),
+  # Discard records from irrelevant tasks
```

Comment:
This is a bit inaccurate. What this is actually doing is ensuring that evaluation metrics don't leak into the training data. Think of it like this: The model shouldn't see/memorize the answers to the final exam. I think
- "Discard records from the evaluation metrics"
- "Prevent test set leakage"
- "Prevent test set contamination"

I'll let you decide on what you think is the best description is.

Reply:
Ah I see, thanks for correcting my understanding. Revised.